_posts/en/newsletters/2021-03-10-newsletter.md

```diff
@@ -0,0 +1,281 @@
+      would still be a several month delay before taproot activated to
+      allow most of the economy to upgrade their nodes.  A [draft
+      implementation][bitcoin core #21377] for this proposal based on
+      Bitcoin Core's existing [BIP9][] implementation was also written
+      by Anthony Towns.  Andrew Chow created an [alternative draft
+      implementation][bitcoin core #21392] based on the previously
+      proposed BIP8 implementation.
+
+    It seemed unlikely any of the proposals would ever become almost
+    everyone's first choice, but it appeared that a large number of
+    people were [willing to accept][folkson gist] the short-duration
+    attempt under the name *Speedy Trial*.  There were still a few
+    concerns with it, including:
+
+    - *Could be co-opted for mandatory activation:* even though the
+      proposal explicitly encourages making other activation attempts if
+      miners don't quickly signal sufficient support for taproot, a
+      concern was [expressed][corallo not speedy enough] that it could
+      be co-opted by a group of users seeking fast mandatory activation,
+      although it was [noted][##taproot-activation log 3/5] that no
+      group has previously expressed the desire to attempt mandatory
+      activation on such a dangerously short timeline.
+
+    - *Using time-based or height-based parameters:* the proposal
+      describes the tradeoffs between setting its `start`, `timeout`,
+      and `minimum_activation` parameters using either times (based on
+      the median of the previous 11 blocks) or block heights.  Using
+      times would result in the smallest and easiest to review patch to
+      Bitcoin Core.  Using heights would provide a bit more
+      predictability, especially for miners, and would be compatible
+      with other attempts using BIP8.
+
+    - *Myopic:* there was [concern][russell concern] that the proposal
+      is too focused on the short term: "Speedy Trial fully prepares for
+      the (likely) case where miners activate taproot, but it does
+      nothing to codify lessons learned from Segwit's failure to
+      activate in a timely manner.  We have an opportunity with the
+      activation of taproot to create a template for future activations
+      that will clearly define the roles and responsibilities for
+      developers, miners, merchants, investors, and end users in all the
+      ways an activation can progress, not just the best-case outcomes;
+      in particular enabling and enshrining the final arbiter role held
+      by Bitcoin's economic users.  Defining this will only get more
+      difficult in the future, both because we'll only do so when we're
+      already in crisis, and because Bitcoin's growth means future
+      agreement will need to be done at greater scale and so with
+      greater difficulty."  <!-- statement written by me trying to
+      summarize Rusty Russell's concerns, then revised by him -->
```

Comment:
If we have to make a change, I'd rather we drop the quotation marks than attribute Rusty as (1) we don't attribute any of the other criticisms to specific people; (2) this was my summary of the situation that Rusty just edited (which also justifies our using it without quotes if necessary), I'm not sure Rusty would phrase it that way on his own.

If we want, we can link to the log where it was discussed, http://gnusha.org/taproot-activation/2021-03-08.log , unfortunately that log site doesn't allow linking to specific lines, which is why I tend to avoid using it when there's been a busy day or an alternative is available.